### PR TITLE
Test configure app resources

### PR DESCRIPTION
**Pull request by maasg**
Adds an integration test to validate the update of resources for an Akka streamlet through a configuration file.

PS: If you know a better way to deal with the parsing of the kubectl/go output, pls let me know.
I tried the Json way but `kubectl -o json-template` does not output JSON but go structs as well :-(

### CODE REVIEW

File: integration-tests/itest/deploy_test.go [hunk 1]
```diff
@@ -143,12 +145,48 @@ var _ = Describe("Application deployment", func() {
 			close(done)
 		}, LongTimeout)
 
-		It("should have configured a flink streamlet", func(done Done) {
+		XIt("should have configured a flink streamlet", func(done Done) {
```
Comment (RayRoestenburg) [inline integration-tests/itest/deploy_test.go:148]: Ok. so we keep this as pending for now so it does not break stuff, LGTM.

File: integration-tests/itest/deploy_test.go [hunk 2]
```diff
@@ -143,12 +145,48 @@ var _ = Describe("Application deployment", func() {
...
+			Expect(someAkkaPod).NotTo(Equal(nil))
+			podRes, err := kubectl.GetPodResources(swissKnifeApp.Name, someAkkaPod.Pod)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Reconfigure a single Akka streamlet")
+			err = cli.Configure(swissKnifeApp, UpdateAkkaProcessResourcesFile)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Wait for the deployment of the new configuration")
+			oneSecond, _ := time.ParseDuration("5s")
```
Comment (RayRoestenburg) [inline integration-tests/itest/deploy_test.go:169]: fiveSeconds ;-)

File: integration-tests/itest/deploy_test.go (hunk 1 above)
Comment (maasg) [reply integration-tests/itest/deploy_test.go:148]: Yeah. It produces a random failure. -- we should test it manually from time to time, though.

**Comment by maasg** [comment]
@yuchaoran2011 I tried all options. `-o json` returns the complete resource in JSON. `-o jsonpath` returns the addressed element, but in go template format :shrug: (https://github.com/kubernetes/kubernetes/issues/16707) 
After the release pressure, I'll look into the go client for k8s as a better structured option.

**Pull request merged**